LeaveApplication.Model/ViewModel/ResponseModel/UserInformationResponseModel.cs

```diff
@@ -10,7 +10,6 @@ public class UserInformationResponseModel
         public string Email { get; set; }
         public string PhoneNumber { get; set; }
         public DateTime CreatedDate { get; set; }
-        public DateTime UpdatedTime { get; set; }
         public DateTime UpdatedDate { get; set; }
```

Comment:
We might need to make updatedTime nullable

Reply:
This is now handled